integration/kubernetes/confidential/lib.sh

```diff
@@ -83,6 +83,7 @@ checkout_doc_repo_dir() {
 
 kubernetes_create_ssh_demo_pod() {
 	checkout_doc_repo_dir
+	[ "${AA_KBC:-}" == "eaa_kbc" ] && sed -i 's#katadocker/ccv0-ssh#katadocker/ssh-demo-eaa-kbc#g' "${doc_repo_dir}/demos/ssh-demo/k8s-cc-ssh.yaml"
```

Comment:
Looks a bit hackish.
OK for this PR, but maybe consider a real template file with replacement in a future iteration, like we do for config files?

Reply:
I think we're interested it moving this set of container images to quay.io at some point, so we should remember to consider the templating at that time.